### PR TITLE
Fix for #736: Expect response code 200 for Selenium containers version >= 3

### PR DESCRIPTION
**Pull request by georgwolf**
#### Short description of what this resolves:
Docker drone is expecting the wrong responseCode for Selenium from version 3 onwards

#### Changes proposed in this pull request:

- Retrieve Selenium major version from image tag
- For version >= 3 and as default (e.g. to cover "latest" tag) expect 200 response code
- For versions < 3 expect 403 response code

**Fixes**: #736

### CODE REVIEW

**Comment by lordofthejars** [comment]
There are some build failures https://travis-ci.org/arquillian/arquillian-cube/builds/254850686?utm_source=github_status&utm_medium=notification

**Comment by georgwolf** [comment]
Sorry, forgot to put the WIP in the title

**Comment by lordofthejars** [comment]
No worries, and thanks for the contribution.

**Comment by georgwolf** [comment]
Not really sure what to do in the case of custom browser images: 

- Either add a cube drone configuration property for setting the expected response code
- or add a cube drone configuration property setting the selenium version the image is based on
- or take Selenium version from classpath

Any preferences there?

File: docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java [hunk 1]
```diff
@@ -159,8 +159,19 @@ private static void setDefaultSeleniumCubeProperties(CubeContainer cubeContainer
 
         Await await = new Await();
         await.setStrategy("http");
-        // Doing an http request to selenium node returns a 403 if Jetty is up and running
-        await.setResponseCode(403);
+
+        // Selenium from 3.x onwards returns 200 if started
+        int expectedResponseCode = 200;
```
Comment (lordofthejars) [inline docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java]: Can you add all calculation of `expectedResponseCode ` calculation into  a method?

**Comment by lordofthejars** [comment]
I think that this way _Selenium version from classpath_ would be the best one here, also because we already have this logic implemented.

**Comment by georgwolf** [comment]
Always using the version from class path now to be consistent and extracted to separate method.
Tested for Selenium 3.x by explicitly picking 3.4.0 in pom.xml and adjusting SeleniumContainersTest appropriately.

**Comment by lordofthejars** [comment]
That's great!!! It looks really awesome. Thank you very much for your contribution.

**Pull request merged**